src/server_manager/web_app/app.ts

```diff
@@ -79,13 +85,16 @@ type DigitalOceanServerRepositoryFactory = (session: digitalocean_api.DigitalOce
 export class App {
   private digitalOceanRepository: server.ManagedServerRepository;
   private selectedServer: server.Server;
-  private runningServer: server.Server;
+  private serverBeingCreated: server.ManagedServer;
+  private displayServerBeingCreated: DisplayServer;
+  private serverDisplayTable: ServerDisplayTable = {};
```

Comment:
I'm not convinced this helps us much and there is a bunch of code required to manage it. How would you implement this class without this function? Would it improve the class?

Reply:
I have removed the table in fb762f6 and now query the repositories. I think it makes the code more maintainable as you pointed out.